backend-flask/README.txt

```diff
@@ -0,0 +1,25 @@
+VENV
+To install venv enter the following command:
+
+$ python3 -m venv venv
```

Comment:
what were we using for the website? do we need virtual envs? Just asking because I know the flask tutorials seem to have a thing for them like a Texan has a thing for barbecues

Reply:
I was talking with people and venv is apparently necessary